src/components/MapImageSlider.vue

```diff
@@ -2003,13 +2003,17 @@ $polygon: '~@/assets/images/polygon.png';
     width: 20px;
     display: inline-block;
 }
+#georgiaSlider  {
+  margin-top: 50px;
+}
 #cartogram-svg {
   width:150%;
   height: auto;
 
   #title-svg {
-    color: $darkGray;
-    font-weight: bold;
+    fill: rgb(139, 139, 139);
+    font-style: italic;
+    font-size: 10px;
```

Comment:
if you've got the style here, we should be able to delete it out of the R code entirely now

Reply:
yeah totally, I thought we might prefer to keep it in the R code for full reproducibility